### PR TITLE
[scripts/install] support for buster

### PR DESCRIPTION
**Pull request by arthurlutz**

### CODE REVIEW

**Comment by arthurlutz** [comment]
Tested successfully in a VM. Haven't tested upgrade from stretch.

**Comment by liberodark** [comment]
is better is merge on other branch buster

**Comment by arthurlutz** [comment]
I see that the jellyfin-ffmpeg .deb package is released here https://github.com/YunoHost-Apps/jellyfin_ynh/releases/ too. What's the difference with the package published by the project ? Here https://github.com/jellyfin/jellyfin-ffmpeg/releases/ 

Would it be interesting to use their debian repo as source ? https://repo.jellyfin.org/ (see https://jellyfin.org/downloads/ )

File: conf/jellyfin-ffmpeg-amd64.src [hunk 1]
```diff
@@ -1,5 +1,5 @@
-SOURCE_URL=https://github.com/jellyfin/jellyfin-ffmpeg/releases/download/v4.2.1-7/jellyfin-ffmpeg_4.2.1-7-stretch_amd64.deb
-SOURCE_SUM=04ed00a1121832fe7c32f78fe9cbbebb0a36ef37c9e2689a75b55efa31ec2b73d670ddcb613d01b14721f3db98e409eca208ee425f4853b343155f06906e6e08
+SOURCE_URL=https://github.com/jellyfin/jellyfin-ffmpeg/releases/download/v4.2.1-7/jellyfin-ffmpeg_4.2.1-7-buster_amd64.deb
```
Comment (arthurlutz) [inline conf/jellyfin-ffmpeg-amd64.src:1]:
```suggestion
SOURCE_URL=https://repo.jellyfin.org/releases/server/debian/versions/jellyfin-ffmpeg/4.3.1-1/jellyfin-ffmpeg_4.3.1-1-buster_amd64.deb
```

File: conf/jellyfin-ffmpeg-amd64.src [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+SOURCE_SUM=da83f46fb772fa9397144fe206121854f59450a62315299b838d8c48a6384efd6bc161db31752e52b3583f89f91c16cbeba117741f43e07496afffe3e0df07f1
```
Comment (arthurlutz) [inline conf/jellyfin-ffmpeg-amd64.src:2]:
```suggestion
SOURCE_SUM=4eb7d4486b9e729539c0721838a41f6d19f40698cbcd9425017c220909e4ee08
```

File: conf/jellyfin-ffmpeg-amd64.src [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
 SOURCE_SUM_PRG=sha512sum
```
Comment (arthurlutz) [inline conf/jellyfin-ffmpeg-amd64.src:3]:
```suggestion
SOURCE_SUM_PRG=sha256sum
```

File: conf/jellyfin-ffmpeg-arm64.src [hunk 4]
```diff
@@ -1,5 +1,5 @@
-SOURCE_URL=https://github.com/YunoHost-Apps/jellyfin_ynh/releases/download/10.6.0/jellyfin-ffmpeg_4.3.1-1-stretch_arm64.deb
-SOURCE_SUM=17ca53a20ab0668f2d5762f0a183c6bcf192cc901e3223e670d7f5c9d7d0c6c7da432f0cdb2fe98108f84d4cb3ac775f84cf5816b8d43d2080a280dab8ba6c32
+SOURCE_URL=https://github.com/jellyfin/jellyfin-ffmpeg/releases/download/v4.2.1-7/jellyfin-ffmpeg_4.2.1-7-buster_arm64.deb
```
Comment (arthurlutz) [inline conf/jellyfin-ffmpeg-arm64.src:1]:
```suggestion
SOURCE_URL=https://repo.jellyfin.org/releases/server/debian/versions/jellyfin-ffmpeg/4.3.1-1/jellyfin-ffmpeg_4.3.1-1-buster_arm64.deb
```

File: conf/jellyfin-ffmpeg-arm64.src [hunk 5]
```diff
@@ -1,5 +1,5 @@
...
+SOURCE_SUM=c089cfd2b31b31114faed00aa7d8df3f44ed729d7c3119d8c04da92507c6fb009903364a4e52fbc14e036f466caa40758eff7d3e0506d9720e10c2aee484810f
```
Comment (arthurlutz) [inline conf/jellyfin-ffmpeg-arm64.src:2]:
```suggestion
SOURCE_SUM=cb1ae9b1611a287ac7b02d3ca7899050890961852c56e90a144b5d5077fa629d
```

File: conf/jellyfin-ffmpeg-arm64.src [hunk 6]
```diff
@@ -1,5 +1,5 @@
...
 SOURCE_SUM_PRG=sha512sum
```
Comment (arthurlutz) [inline conf/jellyfin-ffmpeg-arm64.src:3]:
```suggestion
SOURCE_SUM_PRG=sha256sum
```

File: conf/jellyfin-ffmpeg-armhf.src [hunk 7]
```diff
@@ -1,5 +1,5 @@
-SOURCE_URL=https://repo.jellyfin.org/releases/server/debian/versions/jellyfin-ffmpeg/4.2.1-2/jellyfin-ffmpeg_4.2.1-2-stretch_armhf.deb
-SOURCE_SUM=a08a4e0db96e05e4382c5c9bba435d658ba4a8a368dba9c06e816bf7ce4bbeee01481aa92919aceaf1f0c1a337356c8c1e1136c8f5b04e05edbe1a9e883df88c
+SOURCE_URL=https://github.com/jellyfin/jellyfin-ffmpeg/releases/download/v4.2.1-7/jellyfin-ffmpeg_4.2.1-7-buster_armhf.deb
```
Comment (arthurlutz) [inline conf/jellyfin-ffmpeg-armhf.src:1]:
```suggestion
SOURCE_URL=https://repo.jellyfin.org/releases/server/debian/versions/jellyfin-ffmpeg/4.3.1-1/jellyfin-ffmpeg_4.3.1-1-buster_armhf.deb
```

File: conf/jellyfin-ffmpeg-armhf.src [hunk 8]
```diff
@@ -1,5 +1,5 @@
...
+SOURCE_SUM=1f534b0a3d34b17348f00d5e2e5d0b2a584c34f8cc222e9fad93324d8e68029d7b60595890a4c69810bc8c9dae793e1f7e5918f2012170f483c3aca6ca399ab5
```
Comment (arthurlutz) [inline conf/jellyfin-ffmpeg-armhf.src:2]:
```suggestion
SOURCE_SUM=7fef8aa370deda95fe92d9ffa14f5788f4fd13ce0a5285eb79a150dcadd06c9e
```

File: conf/jellyfin-ffmpeg-armhf.src [hunk 9]
```diff
@@ -1,5 +1,5 @@
...
 SOURCE_SUM_PRG=sha512sum
```
Comment (arthurlutz) [inline conf/jellyfin-ffmpeg-armhf.src:3]:
```suggestion
SOURCE_SUM_PRG=sha256sum
```

**Review by arthurlutz — COMMENTED** [review]
Upgrade to 4.3.1-1

File: scripts/install [hunk 10]
```diff
@@ -79,7 +79,7 @@ ynh_print_info "Installing dependencies..."
 ###		- As well as the section "REINSTALL DEPENDENCIES" in the restore script
 ###		- And the section "UPGRADE DEPENDENCIES" in the upgrade script
 
-ynh_install_app_dependencies at libass5 libbluray1 libdrm2 libmp3lame0 libopus0 libtheora0 libva-drm1 libva-x11-1 libva1 libvdpau1 libvorbis0a libvorbisenc2 libx264-148 libx265-95 libzvbi0 libwebpmux2
+ynh_install_app_dependencies at libass9 libbluray2 libdrm2 libmp3lame0 libopus0 libtheora0 libva-drm2 libva-x11-2 libva2 libvdpau1 libvorbis0a libvorbisenc2 libx264-155 libx265-165 libzvbi0
```
Comment (arthurlutz) [inline scripts/install:82]:
```suggestion
ynh_install_app_dependencies at libass9 libbluray2 libdrm2 libmp3lame0 libopus0 libtheora0 libva-drm2 libva-x11-2 libva2 libvdpau1 libvorbis0a libvorbisenc2 libx264-155 libx265-165 libzvbi0 ocl-icd-libopencl1
```

**Comment by arthurlutz** [comment]
Since clicking upgrade on a buster breaks jellyfin by trying to install the stretch version. I upgraded my buster patch. 

Tested on amd64 and armhf (pi). The conflicts file are because without an answer to my comment in August, I think it's probably a better idea to point to the upstream .deb files instead of a (re-published?) .deb file on this project.

**Comment by liberodark** [comment]
A buster version is coming later.
When have time im go to make a dedicated buster version.

**Comment by liberodark** [comment]
Now master is for YNH 4.x :D

**Pull request closed without merging**